README.md

```diff
@@ -1,6 +1,14 @@
 # Kubernetes Cluster API Provider BYOH(Bring your own host)
-[![lint](https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/actions/workflows/lint.yml/badge.svg)](https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/actions/workflows/lint.yml)
-[![test](https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/actions/workflows/main.yml/badge.svg)](https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/actions/workflows/main.yml)
+<p align="center">
```

Comment:
I don't have a personal preference but most of the OSS projects keep it left aligned.
https://github.com/trending/go

Reply:
@shamsher31 we can do that too, don't have any strong prefs there, was just trying to align with other CAPI-Providers